src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String [] str = new String[0];
```

Comment:
```suggestion
        String[] str = new String[0];
```
let's fix this everywhere please

Reply:
done